strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    es = Podría ser muy caro con ciertos planes o con itinerancia de datos.
+    pl = Może to być kosztowne przy niektórych planach taryfowych lub w roamingu.
+    nl = Met sommige abonnementen of bij roaming kan dit behoorlijk duur zijn.
+    pt = Isto pode ser significativamente caro, com alguns planos ou se roaming.
+
+  [location_services_disabled_header]
+    comment = iOS Заголовок Диалога при отключенном пермишине на поиск геопозиции
+    en = Location services are disabled
+    ru = Определение местоположения отключено
+    fr = L'identification de la localisation est désactivée
+    da = Placeringsservices er slået fra
+    id = Identifikasi lokasi dinonaktifkan
+    ko = 위치 식별 사용할 수 없음
+    sv = Platsidentifiering avaktiverad
+    tr = Konum tanımlama devre dışı bırakıldı
+    uk = Ідентифікацію місцезнаходження відключено
+    vi = Chức năng nhận biết địa điểm bị tắt
+    hu = Helyzetmeghatározás kikapcsolva
+    de = Standort-Identifizierung ist deaktiviert
+    fi = Locatie-identificatie is uitgeschakeld
+    cs = Určení polohy je zakázáno
+    it = La geolocalizzazione è disabilitata
+    nb = Lokalisering er deaktivert
+    zh-Hant = 地點定位被禁用
+    zh-Hans = 地点定位被禁用
+    th = ปิดการใช้งานการระบุตำแหน่งที่ตั้งแล้ว
+    ja = 位置情報認識機能が無効になっています
+    ro = Geolocalizarea este dezactivată
+    ar = تم تعطيل تحديد الموقع
+    sk = Identifikácia lokality je zablokovaná
+    es = La identificación de la ubicación está desactivada
+    pl = Identyfikacja lokalizacji jest wyłączona
+    nl = Locatie-identificatie is uitgeschakeld
+    pt = A identificação do local está desativada
+    he = זיהוי המיקום אינו מופעל
+
+  [location_services_disabled_1]
+    comment = iOS Текст Диалога при отключенном пермишине на поиск геопозиции
+    en = 1. Open Settings
+    ru = 1. Откройте Настройки
+    fr = 1. Ouvrir Paramètres
+    da = 1. Åbn Indstillinger
+    id = 1. Buka Pengaturan
+    ko = 1. 설정 열기
+    sv = 1. Öppna inställningar
+    tr = 1. Ayarları Açın
+    uk = 1. Відкрийте «Налаштування»
+    vi = 1. Mở Thiết lập
+    hu = 1. Nyissa meg a Beállításokat
+    de = 1. Einstellungen öffnen
+    fi = 1. Instellingen openen
+    cs = 1. Otevřete Nastavení
+    it = 1. Apri le impostazioni
+    nb = 1. Åpne Innstillinger
+    zh-Hant = 1. 打開設置
+    zh-Hans = 1. 打开设置
+    th = 1. เปิดการตั้งค่า
+    ja = 1. 設定を開きます
+    ro = 1. Deschideți setările
+    ar = 1. افتح الإعدادات
+    sk = 1. Otvorte Nastavenia
+    es = 1. Abre los ajustes
+    pl = 1. Uruchom ustawienia
+    nl = 1. Instellingen openen
+    pt = 1. Abrir configurações
+    he = 1. פתחו את "הגדרות" (Settings)
+
+  [location_services_disabled_2]
+    en = 2. Tap Location
+    ru = 2. Нажмите Геопозиция
+    fr = 2. Appuyer sur Localisation
+    da = 2. Tryk på Placering
+    id = 2. Ketuk Lokasi
+    ko = 2. 위치 누르기
+    sv = 2. Tryck på Plats
+    tr = 2. Konuma Dokunun
+    uk = 2. Клацніть «Місцезнаходження»
```

Comment:
Надо подключать кого-то более понимающего в правилах переводов, сложно сказать нарочно ли это или ошибка.
> On Apr 26, 2016, at 4:37 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61085914>:
> 
> > +    es = 1. Abre los ajustes
> > +    pl = 1. Uruchom ustawienia
> > +    nl = 1. Instellingen openen
> > +    pt = 1. Abrir configurações
> > +    he = 1. פתחו את "הגדרות" (Settings)
> > +
> > +  [location_services_disabled_2]
> > +    en = 2. Tap Location
> > +    ru = 2. Нажмите Геопозиция
> > +    fr = 2. Appuyer sur Localisation
> > +    da = 2. Tryk på Placering
> > +    id = 2. Ketuk Lokasi
> > +    ko = 2. 위치 누르기
> > +    sv = 2. Tryck på Plats
> > +    tr = 2. Konuma Dokunun
> > +    uk = 2. Клацніть «Місцезнаходження»
> И тут где-то есть кавычки, где-то нет. Где-то одинарные, где-то двойные.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61085914>